cmd/xl-v1-common.go

```diff
@@ -81,7 +82,7 @@ func (xl xlObjects) isObject(bucket, prefix string) (ok bool) {
 	// quorum intentionally, but rely on the default case scenario. Actual quorum
 	// verification will happen by top layer by using getObjectInfo() and will be
 	// ignored if necessary.
-	readQuorum := len(storageDisks) / 2
+	readQuorum := int(math.Ceil(float64(len(storageDisks)) / 2.0))
```

Comment:
```suggestion
	readQuorum := readQuorum(len(storageDisks))
```
(or just inline below)